### PR TITLE
mdcat: new port

### PR DESCRIPTION
**Pull request by herbygillot**
New port for [mdcat](https://github.com/lunaryorn/mdcat)

###### Tested on

macOS 10.15.5 19F101
Xcode 11.5 11E608c

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -d install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: textproc/mdcat/Portfile [hunk 1]
```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        lunaryorn mdcat mdcat-0.19.0
```
Comment (reneeotten) [inline textproc/mdcat/Portfile:7]: this is incorrect usage, now the version will be `mdcat-0.19.0`; instead you should use `github.setup        lunaryorn mdcat 0.19.0 mdcat-`.  The PG does `proc github.setup {gh_author gh_project gh_version {gh_tag_prefix ""} {gh_tag_suffix ""}}`.

Comment (herbygillot) [reply textproc/mdcat/Portfile]: You're right - wasn't thinking here.  Change made.

**Comment by reneeotten** [comment]
thanks @herbygillot !

**Comment by herbygillot** [comment]
Thank you @reneeotten

**Pull request merged**